tests/upgrades-images-unsigned/upgrade_test.go

```diff
@@ -19,22 +20,22 @@ var _ = Describe("cOS Upgrade tests - Images unsigned", func() {
 	})
 	Context("After install", func() {
 		When("images are not signed", func() {
-			It("fails to upgrade to a version which is not signed", func() {
-				out, err := s.Command("cos-upgrade --docker-image raccos/releases-opensuse:cos-system-0.5.0")
-				Expect(err).To(HaveOccurred())
-				Expect(out).Should(ContainSubstring("No valid trust data"))
+			It("upgrades to latest available (master) and reset", func() {
```

Comment:
I guess this should try to upgrade to 0.5.0, but if it tries to pull an earlier one it will fail...

Reply:
as far as I understand, master should be on 0.5.0 already and it will try to update to whatever is latest, which should always be >= 0.5.0 as you pushed that image no?

Im more worried about the other test below as I'll have to remove it, there is no point into changing into a unsigned release branch anymore as the main branch (release-opensuse) is already unsigned....